dpnp/tests/test_histogram.py

```diff
@@ -587,19 +587,31 @@ def test_different_bins_amount(self, bins_count):
     @pytest.mark.parametrize(
         "array",
         [[1, 2, 3], [1, 2, 2, 1, 2, 4], [2, 2, 2, 2]],
-        ids=["[1, 2, 3]", "[1, 2, 2, 1, 2, 4]", "[2, 2, 2, 2]"],
+        ids=["size=3", "size=6", "size=4"],
     )
-    @pytest.mark.parametrize(
-        "minlength", [0, 1, 3, 5], ids=["0", "1", "3", "5"]
-    )
-    def test_bincount_minlength(self, array, minlength):
+    @pytest.mark.parametrize("minlength", [0, 1, 3, 5])
+    def test_minlength(self, array, minlength):
         np_a = numpy.array(array)
         dpnp_a = dpnp.array(array)
 
         expected = numpy.bincount(np_a, minlength=minlength)
         result = dpnp.bincount(dpnp_a, minlength=minlength)
         assert_allclose(expected, result)
 
+    # TODO: uncomment once numpy 2.3.0 is released
+    # @testing.with_requires("numpy>=2.3")
+    # @pytest.mark.parametrize("xp", [dpnp, numpy])
+    @pytest.mark.parametrize("xp", [dpnp])
```

Comment:
Probably it is better to try something like:
```
@pytest.mark.parametrize("xp", [dpnp, pytest.param(numpy, marks=pytest.mark.xfail(condition=..., reason=..., strict=True)]
```
or
```
```
@pytest.mark.parametrize("xp", [dpnp, pytest.param(numpy, marks=pytest.mark.skipif(condition=..., reason=...)]
```
```

Reply:
Thank you for the hint, the explicit `pytest.mark.xfail` is added instead of TODO comment.